dbt_platform_helper/commands/pipeline.py

```diff
@@ -33,15 +35,22 @@ def pipeline():
 
 
 @pipeline.command()
-def generate():
+@click.option(
+    "--terraform-platform-modules-version",
+    help=f"""Override the default version of terraform-platform-modules with a specific version or branch. 
+    (Default version is '{DEFAULT_TERRAFORM_PLATFORM_MODULES_VERSION}').""",
+    default=DEFAULT_TERRAFORM_PLATFORM_MODULES_VERSION,
+)
+def generate(terraform_platform_modules_version):
     """Given a platform-config.yml file, generate environment and service
     deployment pipelines."""
     pipeline_config = load_and_validate_platform_config()
 
     no_codebase_pipelines = CODEBASE_PIPELINES_KEY not in pipeline_config
     no_environment_pipelines = ENVIRONMENTS_KEY not in pipeline_config
+    no_terraform_environment_pipelines = ENVIRONMENT_PIPELINES_KEY not in pipeline_config
```

Comment:
I would be inclined to reverse the extra word as we'll soon be losing all the legacy stuff. Perhaps actually reverse the boolean too, then use it in all the other places we perform a the same check...
```suggestion
    has_codebase_pipelines = CODEBASE_PIPELINES_KEY in pipeline_config
    has_legacy_environment_pipelines = ENVIRONMENTS_KEY in pipeline_config
    has_environment_pipelines = ENVIRONMENT_PIPELINES_KEY in pipeline_config
```

Reply:
Thanks, I am inclined to agree and I'll make the update.